hipparchus-core/src/main/java/org/hipparchus/util/FieldBlendable.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Please add your copyright here.

Reply:
Hi @maisonobe, 

At first, I thought that I had forgotten (again) to add the Hipparchus copyright but i'm glad I did not. Hence, you surely talk about the contributing license agreement that is mentioned in the contribution guidelines but it appears the link is dead (error 404) :sweat:  : https://hipparchus.org/licenses/#clas.

Reply:
No, I am talking about the first line in the header.
It is currently empty but should refer to the original owner of intellectual property rights. I guess it is CS GROUP in your case.

Reply:
Ok, I replaced the copyright with the CS Group one from Orekit. I thought leaving both of them was redundant as 95% of the text was identical but it was maybe a mistake ?

Reply:
No, this is not correct.
What I wanted was just to replace the *first* line that holds the copyright.
The rest of the header should have remained as is if you want the *Hipparchus* project to be able to publish this code.
So as a summary, CS GROUP holds the copyright (first line) and grants to the Hipparchus project a license to publish it.

Reply:
Okay so  i reused the old license and added a CS Group copyright on the first line.